### PR TITLE
feat(operator): add namespace to grafanaDashboard

### LINKED ISSUE #138: [Feature]: Install Grafana dashboard in the namespace where kube-prometheus is located
### Checklist

- [X] I've searched for similar issues and couldn't find anything matching
- [X] I've discussed this feature request in the [K8sGPT Slack](https://k8sgpt.slack.com) and got positive feedback

### Is this feature request related to a problem?

Yes

### Problem Description

Without making any modifications, the dashboard will be installed in the namespace where k8sgpt is located. If the namespaces of kube-prometheus and k8sgpt are different, the dashboard will not work. 

### Solution Description

Adding namespace options in helm

### Benefits

Can make the Grafana dashboard work smoothly.

### PR DESCRIPTION
**Pull request by MateSousa**
Closes #138 

## 📑 Description
The `namespace` field has been added to the `grafanaDashboard` section of the `values.yaml` file. This allows the Grafana dashboard to be deployed to a specific namespace, which improves organization and separation of concerns.

## ✅ Checks
- [x] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed

### CODE REVIEW

**Comment by arbreezy** [comment]
hey @MateSousa  thanks for the PR and the interest in the project.

I believe we want something similar to what we define in prometheus stanza, whereby if `namespace` is not defined in values.yaml we deploy the Grafana dashboard's CM in the same namespace where we deploy the operator

**Comment by MateSousa** [comment]
@arbreezy i guess it would be like that, right?

**Review by AlexsJones — DISMISSED** [review]

**Review by arbreezy — CHANGES_REQUESTED** [review]

File: chart/operator/templates/grafana-k8sgpt-dashboard.yaml [hunk 1]
```diff
@@ -11,6 +11,12 @@ metadata:
     app.kubernetes.io/part-of: k8sgpt-operator
   {{- include "chart.labels" . | nindent 4 }}
   name: k8sgpt-overview
+spec:
+  {{- if .Values.grafanaDashboard.namespace }}
+  namespaceSelector:
```
[the remark this answers is not in the document]
Comment (MateSousa) [reply chart/operator/templates/grafana-k8sgpt-dashboard.yaml:16]: Ops, i'm going to fix it

**Review by arbreezy — APPROVED** [review]

**Comment by arbreezy** [comment]
@MateSousa thank you for your contribution

**Pull request merged**